4week/src/components/Sidebar.tsx

```
@@ -1,88 +1,148 @@
-      </div>
-
-      {/* 배경 클릭 시 닫힘 */}
-      {open && <div className="fixed inset-0" onClick={() => setOpen(false)} />}
-    </>
-  );
+import { Link } from "react-router-dom";
+import useDeleteUsers from "../hooks/mutations/useDeleteUsers";
+import { Modal } from "./Modal";
+
+interface SidebarProps {
+  isOpen: boolean;
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)